src/github-issues-prs.ts

```diff
@@ -104,10 +104,37 @@ export class GitHubIssuesPrsProvider implements TreeDataProvider<TreeItem> {
 			return false;
 		}
 
-		// take first one
-		await open(getGitHubUrl(remotes[0].url) + '/issues/new');
+		let urls: string[] = remotes.map(remote => remote.url);
 
-		return true;
+		window.showQuickPick(
+			Promise.resolve(urls),
+			{
+				placeHolder: 'Select the remote you want to create an issue on'
+			}
+		).then((remote: string|null) => {
+			if (!remote) {
+				return;
+			}
+
+
+			// get my object back
+			const selectedRemote = remotes.find(x => x.url === remote);
```

Comment:
Actually, instead of passing 'urls' above you could pass an array of QuickPickItem with a label that is 'owner/repo' and an additional property 'remote'. That will give a nicer label (and more flexibility). You'll get the selected QuickPickItem as the result and can directly use the remote from that.